### PR TITLE
Add technical article: Using Aspose.Imaging for Java in Kotlin

### PR DESCRIPTION
**Pull request by anms23**
IMAGINGJAVA-8614: article about the possibility of using Aspose.Imaging for Java in Kotlin

### CODE REVIEW

File: java/developer-guide/technical-articles/using-imaging-in-kotlin/_index.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+url: /java/using-imaging-in-kotlin/
+description: How to use Aspose.Imaging for Java in Kotlin applications for desktops and servers
+keywords: Kotlin applications, Aspose.Imaging for Java, export svg to png in Kotlin, create images in Kotlin, modify images in Kotlin, load images in Kotlin, save images in Kotlin, crop image in Kotlin, merge images in Kotlin
+---
+
+## **Using Aspose.Imaging for Java in Kotlin Applications**
+
+<p align='justify'>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+In the dynamic landscape of software development, the synergy between programming languages is a powerful force. Kotlin, known for its conciseness and interoperability with Java, opens doors for developers seeking to enhance their applications with sophisticated features. The Aspose.Imaging graphic library is applicable for both desktop and server applications. As of version 23.12, it does not support the Android platform, but support for it will be included in the upcoming releases.
```
Comment (ESidenko) [inline java/developer-guide/technical-articles/using-imaging-in-kotlin/_index.md]: Perhaps the better to change "As of version 23.12" with "For now"

**Comment by ESidenko** [comment]
Perhaps the better to change "As of version 23.12" with "For now"

**Review by ESidenko — CHANGES_REQUESTED** [review]
Perhaps the better to change "As of version 23.12" with "For now"

File: java/developer-guide/technical-articles/using-imaging-in-kotlin/_index.md (hunk 1 above)
Comment (anms23) [reply java/developer-guide/technical-articles/using-imaging-in-kotlin/_index.md]: Fixed: https://github.com/aspose-imaging/Aspose.Imaging-Documentation/pull/19/commits/85dd6982e17dc397b4f74a8ea02617218a7ba741

**Review by ESidenko — APPROVED** [review]

**Pull request merged**